src/content/en/fundamentals/getting-started/primers/media-source-extensions.md

```diff
@@ -0,0 +1,323 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Media Source Extensions (MSE) is a JavaScript API that lets you build streams for playback from segments of audio or video.
+
+{# wf_published_on: 2017-02-08 #}
+{# wf_updated_on: 2017-02-08 #}
+
+# Media Source Extensions {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+Media Source Extensions (MSE) is a JavaScript API that lets you build streams
+for playback from segments of audio or video. Although not covered in this
+article, understanding MSE is needed if you want to embed videos in your site
+that does such things as:
+
++  Adaptive streaming, which is another way of saying adapting to device
+   capabilities and network conditions
++  Adaptive splicing
++  Time shifting
++  Control of performance and download size
+
+You can almost think of MSE as a chain. Between the downloaded file and the media
+elements are several layers.
+
++  An `<audio>` or `<video>` element to play the media.
++  A `MediaSource` instance with a `SourceBuffer` to feed the media element.
++  A `fetch()` or XHR call to retrieve media data in a `Response` object.
++  A call to `Response.arrayBuffer()` to feed `MediaSource.SourceBuffer`.
+
+This is illustrated below. 
+
+![Basic MSE data flow](imgs/basic-mse-flow.png)
```

Comment:
Shift this up above the list, change to HTML and add `class="attempt-right"` so that it flows within the doc body. You'll likely need a `<div class="clear-fix"></div>`  before the `In practice, the chain looks like...`